src/gordon_gcp/plugins/enricher.py

```diff
@@ -49,11 +57,96 @@ class GCEEnricher:
     """
     phase = 'enrich'
 
-    def __init__(self, config, success_channel, error_channel):
+    def __init__(self, config, http_client, success_channel, error_channel):
         self.config = config
         self.success_channel = success_channel
         self.error_channel = error_channel
+        self._http_client = http_client
+        self._logger = logging.getLogger('')
+
+    async def _poll_for_ext_ip(self, resource_name, msg_logger):
+        exception = None
+        backoff = 2
```

Comment:
Hmm we should put backoff in the http client eventually 🤔

Reply:
Created a ticket off this.